tasks/modules/compile.js

```diff
@@ -148,7 +148,17 @@ function compileAllFiles(targetFiles, target, task, targetName) {
     }
     // string options
     args.push('--target', task.target.toUpperCase());
-    args.push('--module', task.module.toLowerCase());
+    // check the module compile option
+    var moduleOptionString = task.module.toLowerCase();
```

Comment:
Wouldn't call '' a valid value. Also I've seen friends pass in null e.g https://github.com/TypeStrong/atom-typescript/issues/123#issuecomment-76680705 Which will break lower case call 

If they don't want to specify a module they should remove the option.